src/pages/Popup/Popup.jsx

```diff
@@ -106,7 +106,7 @@ const Popup = () => {
     if (newButtonState) {
       sendMessage({
         action: 'startPlaying',
-        value: 'I want to play events now!',
+        value: document.getElementById("recordString").value,
```

Comment:
attach the input like this:
https://stackoverflow.com/a/58211012/904992

Reply:
updated